ff/Cargo.toml

```diff
@@ -36,4 +36,3 @@ rustc_version = "0.4"
 default = []
 std = [ "ark-std/std", "ark-serialize/std" ]
 parallel = [ "std", "rayon", "ark-std/parallel" ]
-asm = []
```

Comment:
Hm I don't think we should enable it by default, no? It's still unsafe that people might not want.

Reply:
then let me add it back.

Reply:
Resolved.